superset-frontend/src/common/components/InfoTooltip.tsx

```diff
@@ -0,0 +1,80 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React from 'react';
+import { styled } from '@superset-ui/core';
+import Tooltip from 'src/common/components/Tooltip';
+import Icon from 'src/components/Icon';
+
+interface InfoTooltipProps {
+  className?: string;
+  tooltip: string;
+  placement?:
+    | 'bottom'
```

Comment:
I was looking before for a way to import type specifications from `antd` but couldn't find an easy way. It sucks to have to redefine/align the type defs on our side as these could get out of sync as we upgrade antd. Would it make more sense to import this? https://github.com/ant-design/ant-design/blob/master/components/tooltip/index.tsx#L14 

```js
import { InfoTooltipProps } from 'antd/components/tooltip/';
```

Reply:
Mmmh. We may need to start growing a `antdCommonTypes` / `antdTypeHelpers` somewhere for this use case. It's likely that we'll want to reuse these!

Reply:
@rusackas , any thoughts?